apidoc/Titanium/Android/Notification.yml

```
@@ -358,7 +358,7 @@ properties:
         ([Filesystem.externalStorageDirectory](Titanium.Filesystem.externalStorageDirectory)).
 
         Note that this sound property is ignored on Android 8.0 and above. On these OS versions,
-        you must use the [NotificationChannel.sound](Titanium.Android.Titanium.Android.NotificationChannel.sound)
+        you must use the [NotificationChannel.sound]Titanium.Android.NotificationChannel.sound)
```

Both before and after here look incorrect to me...

Is this correct?
```suggestion
        you must use the [NotificationChannel.sound](Titanium.Android.NotificationChannel.sound)
```